### PR TITLE
Fixed jpg and jpeg mime type retrieval for FileMiddleware

### PR DESCRIPTION
**Pull request by sroebert**
The key for retrieving the `jpg` and `jpeg` mime type, was a combined comma separated list of `jpg, jpeg`. This obviously does not work when trying to use `jpg` or `jpeg` as a key. This commit solves that by creating separate entries for both `jpg` and `jpeg`.

### CODE REVIEW

File: Sources/Vapor/Content/MediaType.swift [hunk 1]
```diff
@@ -489,7 +489,8 @@ let mediaTypes = [
     "g3": "image/g3fax", // G3 Fax Image Wikipedia: G3 Fax Image
     "gif": "image/gif", // Graphics Interchange Format Wikipedia: Graphics Interchange Format
     "ief": "image/ief", // Image Exchange Format RFC 1314
-    "jpeg, .jpg": "image/jpeg", // JPEG Image RFC 1314
+    "jpeg": "image/jpeg", // JPEG Image RFC 1314
```
Comment (LoganWright) [inline Sources/Vapor/Content/MediaType.swift]: Is there a situation where we might receive `"jpeg, .jpg"` as well as these other two? If no, then let's get this merged. Thanks for the additions!

Comment (sroebert) [inline Sources/Vapor/Content/MediaType.swift]: Currently there is none. I was thinking of optimising it for that, but it doesn't seem worth it, until there really is a need.

**Comment by natebird** [comment]
This PR looks good to merge.

File: Sources/Vapor/Content/MediaType.swift (hunk 1 above)
Comment (LoganWright) [inline Sources/Vapor/Content/MediaType.swift]: Thanks @sroebert this is getting merged in today. Appreciate the contributions.

**Pull request merged**